tests/core/fbtests/fbtestfixture.cpp

```diff
@@ -153,7 +150,6 @@ void CFBTestFixtureBase::setup(const char* paConfigString){
 }
 
 bool CFBTestFixtureBase::createInterfaceSpec(const char *, SFBInterfaceSpec &paInterfaceSpec) {
```

Comment:
I believe the parameter `paInterfaceSpec` is now unused?

Reply:
yes, fixed.